### ISSUE TITLE
FFMPEG configured to always copy audio, audio format not always supported in MP4 container

### ISSUE THREAD

**Issue by JamesSwift** [photoprism/photoprism#1061]
So I have been getting some failed transcodes and tracked it back to the log:

` Could not find tag for codec wmav2 in stream #1, codec not currently supported in container\nCould not write header for output file #0 (incorrect codec parameters ?): Invalid argument\nError initializing output stream 0:0 -- \nConversion failed!`

According to [this](https://superuser.com/questions/432374/conversion-from-wmv-to-mp4-file-fails-with-ffmpeg), it's because PP is configured to always copy the audio stream, but wma is not supported in mp4 container:

https://github.com/photoprism/photoprism/blob/4939e36ac6b9150aa78c690e69142bf42bde472b/internal/photoprism/convert.go#L340

I'm no expert, but I believe just removing this line will let ffmpeg decide for itself whether to copy or transcode the audio stream.

#1060 

Edit: This doesn't just affect wma, here's a very old video phone clip failing:
`Could not find tag for codec pcm_u8 in stream #1, codec not currently supported in container\nCould not write header for output file #0 (incorrect codec parameters ?): Invalid argument`

**Comment by lastzero** [comment]
Kept it as not all ffmpeg versions have aac support for copyright reasons, so not sure if that causes errors then. Could you do a bit of research for this?

**Comment by JamesSwift** [comment]
Hmm, I wasn't aware of the aac issue. I have ran some tests with ffmpeg inside the PP container, and it converts fine when you explicitly tell it to use aac, or simply don't specify the audio codec. So there wouldn't be an issue for the docker version. I also tested it with the latest ffmpeg build. It's GPLv3 licensed, and seems to include the aac encoder.

What specific scenarios are you worried about?
